loopout.go

```diff
@@ -30,7 +30,7 @@ var (
 
 	// DefaultSweepConfTarget is the default confirmation target we'll use
 	// when sweeping on-chain HTLCs.
-	DefaultSweepConfTarget int32 = 6
+	DefaultSweepConfTarget int32 = 12
```

Comment:
Could set it to 9 and things are ok

```suggestion
	DefaultSweepConfTarget int32 = 9
```